src/server/journal/streamer.h

```diff
@@ -40,13 +45,28 @@ class JournalStreamer : protected BufferedStreamerBase {
     return true;
   }
 
- private:
+  class PeriodicPing {
```

Comment:
sorry, why do we need this class?

Reply:
to separate the timer + pinging the lsn from the JournalStreamer